### PR TITLE
[IMP] stock_picking_invoice_link: Many2many instead of many2one for invoice references

### PR DESCRIPTION
**Pull request by pedrobaeza**
For dropshipping pickings with module _stock_dropshipping_dual_invoice_, you generate
two invoices for the same picking, so the many2one only reflects the last generated
invoice. With this change, now both are linked.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
cc @Tecnativa

File: stock_picking_invoice_link/migrations/8.0.2.0.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, SUPERUSER_ID
+from openupgradelib import openupgrade
```
Comment (eLBati) [inline stock_picking_invoice_link/migrations/8.0.2.0.0/post-migration.py:6]: @pedrobaeza to correctly run these scripts, I only need to install openupgradelib and normally update the module (without using odoo server from openupgrade itself) right?

**Comment by eLBati** [comment]
LGTM :+1: thanks

File: stock_picking_invoice_link/migrations/8.0.2.0.0/post-migration.py (hunk 1 above)
Comment (pedrobaeza) [inline stock_picking_invoice_link/migrations/8.0.2.0.0/post-migration.py:6]: Yeah, that's it. That was the main purpose when extracting openupgradelib from the full OpenUpgrade distribution.

Comment (gurneyalex) [inline stock_picking_invoice_link/migrations/8.0.2.0.0/post-migration.py:6]: @pedrobaeza don't you need to declare the external dependency in **openerp**.py ?

Comment (pedrobaeza) [inline stock_picking_invoice_link/migrations/8.0.2.0.0/post-migration.py:6]: No, because this is only for migrations, not for new installs.

**Comment by gurneyalex** [comment]
:+1:

**Pull request merged**